data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature

```diff
@@ -18,6 +18,8 @@ Feature: Data Import integration tests
     * def defaultJobProfileId = '6f7f3cd7-9f24-42eb-ae91-91af1cd54d0a'
 
   Scenario: FAT-937 Upload MARC file and Create Instance, Holdings, Items.
+    ##actionProfiles, uploadFile, verifyFiles, verifyLogs - we can move these out. -
```

Comment:
Hi @SatishkumarPurswani !
We can remove these comments.

Reply:
Hi @AivarIUsupov - Changes are done.